File: npm/ng-packs/packages/core/src/lib/models/find-tenant-result-dto.ts

export class FindTenantResultDto  {
  success: boolean;
  tenantId?: string;
  name: string;

  constructor(initialValues: Partial<FindTenantResultDto> = {}) {
    for (const key in initialValues) {
      if (initialValues.hasOwnProperty(key)) {
        this[key] = initialValues[key];
      }
    }
  }
}

File: framework/src/Volo.Abp.AspNetCore.Mvc.Contracts/Volo/Abp/AspNetCore/Mvc/MultiTenancy/FindTenantResultDto.cs
using System;

namespace Volo.Abp.AspNetCore.Mvc.MultiTenancy;

[Serializable]
public class FindTenantResultDto
{
    public bool Success { get; set; }

    public Guid? TenantId { get; set; }

    public string Name { get; set; }

    public bool IsActive { get; set; }
}
